apps/cookbook/src/app/showcase/card-showcase/card-showcase.component.html

```diff
@@ -105,14 +105,24 @@ <h2>Background image</h2>
 
 <h2>Flat</h2>
 <p>
-  By default the card has a slightly elevated look with a shadow around its edges. This can be
-  removed by setting the flat input, but be aware that this option should not be used with a card
+  The default card has a slightly elevated look with a shadow around its edges. This can be removed
+  by setting the variant to "flat", but be aware that this option should not be used with a card
   that can be interacted with (clickable, disclosure) as it lacks proper affordance.
 </p>
 <cookbook-example-viewer [html]="flatExample.template">
   <cookbook-card-example-flat #flatExample></cookbook-card-example-flat>
 </cookbook-example-viewer>
 
+<h2>Outlined</h2>
+<p>
+  An outlined variant of a card is shown only with a border around its edges and have no background.
```

Comment:
Consider keeping naming in sync with UX:
```suggestion
  An outline variant of a card is shown only with a border around its edges and have no background.
```